File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/datatabledata/DataTableDataMvpView.java
package com.mifos.mifosxdroid.online.datatabledata;

import com.google.gson.JsonArray;
import com.mifos.mifosxdroid.base.MvpView;

/**
 * Created by Rajan Maurya on 7/6/16.
 */
public interface DataTableDataMvpView extends MvpView {

    void showDataTableInfo(JsonArray jsonElements);

    void showDataTableDeletedSuccessfully();

    void showEmptyDataTable();

    void showFetchingError(int message);

    void showFetchingError(String s);
}

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/datatabledata/DataTableDataMvpView.kt
package com.mifos.mifosxdroid.online.datatabledata

import com.google.gson.JsonArray
import com.mifos.mifosxdroid.base.MvpView

/**
 * Created by Rajan Maurya on 7/6/16.
 */
interface DataTableDataMvpView : MvpView {
    fun showDataTableInfo(jsonElements: JsonArray?)
    fun showDataTableDeletedSuccessfully()
    fun showEmptyDataTable()
    fun showFetchingError(message: Int)
    fun showFetchingError(s: String?)
}